packages/app/cypress/e2e/runner/ui-states.cy.ts

```diff
@@ -0,0 +1,260 @@
+import { runSpec } from './support/spec-loader'
+
+describe('src/cypress/runner ui states', { retries: 0, defaultCommandTimeout: 600000 }, () => {
+  beforeEach(() => {
+    cy.viewport(1000, 1500)
+  })
+
+  it('hooks', () => {
+    runSpec({
+      fileName: 'ui-states/hooks.cy.js',
+    })
+    .then((win) => {
+      return new Promise<void>((resolve) => {
+        win.getEventManager().on('cypress:in:cypress:run:complete', () => {
+          resolve()
+        })
+      }).then(() => {
+        cy.contains('test hooks').should('be.visible')
+        cy.percySnapshot()
+      })
+    })
+  })
+
+  it('nested tests', () => {
+    runSpec({
+      fileName: 'ui-states/nested-tests.cy.js',
+    })
+    .then((win) => {
+      return new Promise<void>((resolve) => {
+        win.getEventManager().on('cypress:in:cypress:run:complete', () => {
+          resolve()
+        })
+      }).then(() => {
+        cy.contains('Nested Tests').should('be.visible')
+        cy.percySnapshot()
+      })
+    })
+  })
+
+  describe('commands', () => {
+    it('commands that dont display in UI', () => {
```

Comment:
maybe a few comments on what commands that don't show up in the UI. I'm guessing this is referring to the spies/stubs & routes